client/src/www/ui_components/app-root.js

```diff
@@ -590,10 +590,11 @@ export class AppRoot extends mixinBehaviors(
         type: Boolean,
         computed: '_computeUseAltAccessMessage(language)',
       },
-      // Feature flag to control whether dark mode is enabled
-      // When set to true, the theme option will appear in the navigation menu
-      // and the app will respect system theme or user theme selection
-      darkModeEnabled: {
+      appearanceSelectionAvailable: {
```

Comment:
Maybe it would be a good idea to get rid of `appearanceSelectionAvailable` because of the conflicting flag in `app.ts`. Enabling this one in `app-root` causes the app to be funky like shown in the previous video.

Reply:
Got it, that makes sense. Releasing as-is sounds like the best move for now.